MSAL/src/native_auth/network/responses/sign_up/MSALNativeAuthSignUpChallengeResponse.swift

```diff
@@ -30,12 +30,11 @@ struct MSALNativeAuthSignUpChallengeResponse: Decodable {
     let interval: Int?
     let challengeTargetLabel: String?
     let challengeChannel: MSALNativeAuthInternalChannelType?
-    let signUpToken: String?
+    let continuationToken: String?
     let codeLength: Int?
 
     enum CodingKeys: String, CodingKey {
         case challengeType, bindingMethod, interval, challengeTargetLabel, challengeChannel, codeLength
-        // API returns signup_token not sign_up_token
-        case signUpToken = "signupToken"
+        case continuationToken = "continuation_token"
```

Comment:
This change is not needed. You can remove the `CodingKeys` enum. This was done because Swift was mapping incorrectly `signUpToken`.

Reply:
Removed